tests/core/utils/test_common.py

```
@@ -146,3 +148,110 @@ def quick():
+            }
+        }
+    }
+    # This is basically: md5("abc")
+    assert common_utils.fuzzer_repeater_perform(arrays) == {
+        "url": ["900150983cd24fb0d6963f7d28e17f72"]
+    }
+
+
+def test_fuzzer_repeater_perform_disallowed_interceptor_raises():
+    arrays = _fuzzer_arrays("os.system")
+    with pytest.raises(ValueError, match="os.system"):
```

Ruff (0.15.10)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)